tests/network/macspoof/conftest.py

```diff
@@ -149,12 +149,14 @@ def linux_bridge_attached_vmb(
 
 @pytest.fixture(scope="class")
 def linux_bridge_attached_running_vma(linux_bridge_attached_vma):
-    return running_vm(vm=linux_bridge_attached_vma, wait_for_cloud_init=True)
+    linux_bridge_attached_vma.wait_for_agent_connected()
```

Comment:
Why do we wait for agent connected and not wait for condition like you did in the other places?

Reply:
I do not understand your comment.

Reply:
Never mind, you can resolve this comment.

Reply:
@Anatw `wait_for_agent_connected` is waiting for the `AgentConnected` condition.